core/os/android/adb/perfetto.go

```diff
@@ -33,7 +33,7 @@ import (
 )
 
 var (
-	ansiRegex = regexp.MustCompile("\u001b\\[\\d+m")
+	AnsiRegex = regexp.MustCompile("\u001b\\[\\d+m")
```

Comment:
Have we considered filing a bug with perfetto to remove this via a command-line option (or better yet they should be removing them automatically if they detect stdout is not going to shell)

Reply:
I'll file a bug. My assumption was that we'd still have to work around it for current android version(s).

Reply:
Internal bug: b/145150184